assets/src/edit-story/components/panels/design/captions/captions.js

```diff
@@ -107,16 +133,15 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
     <SimplePanel
       css={highlight?.showEffect && styles.FLASH}
       name="caption"
-      title={__('Captions', 'web-stories')}
+      title={__('Caption and subtitles', 'web-stories')}
       isPersistable={!highlight}
```

Comment:
I think the whole `highlight` logic and the `buttonRef` can be removed from this panel -- doesn't seem to be relevant anymore for the style.

Reply:
I think it's still used, right? Isn't the highlight used when we click the checklist so that the relevant panel is highlighted?
<img src="https://user-images.githubusercontent.com/22185279/110995076-fb265c80-8336-11eb-934f-782db41ba6a0.gif" width="300px" />

Happy to remove it if we don't need it anymore, but I think we still do 😬

Reply:
Ok, now I see what it's for -- yes, we still need it, thanks! 👍